core/server/data/meta/description.js

```diff
@@ -14,7 +14,7 @@ function getDescription(data, root) {
     } else if (_.contains(context, 'author') && data.author) {
         description = data.author.bio;
```

Comment:
In a similar way to tags, should this perhaps be `data.author.meta_description || data.author.bio`?